deployment/prerequisites.ts

```diff
@@ -47,6 +47,50 @@ export class PrerequisiteStack extends Stack {
     new apigateway.CfnAccount(this, "ApiGatewayRestApiAccount", {
       cloudWatchRoleArn: cloudWatchRole.roleArn,
     });
+
+    const githubProvider = new iam.CfnOIDCProvider(
```

Comment:
Can we not use the L2 construct?  `iam.OpenIdConnectProvider`

Reply:
Fixed.